zerver/openapi/zulip.yaml

```diff
@@ -2809,6 +2862,10 @@ paths:
               schema:
                 allOf:
                 - $ref: '#/components/schemas/JsonError'
+                - properties:
+                    msg:
+                      enum:
+                      - 'Invalid stream id'
```

Comment:
I don't think it makes sense to do an `enum` on `msg`; `code` would be a very reasonable attribute to do this with, however.

Reply:
i18n considerations mean that `msg` will always have a lot of variability.

Reply:
Right, forgot about that. Removed accordingly.